src/pal/src/loader/module.cpp

```diff
@@ -672,11 +672,12 @@ PAL_RegisterLibraryDirect(
 HINSTANCE
 PALAPI
 PAL_RegisterModule(
-    IN LPCSTR lpLibFileName)
+    IN LPCSTR lpLibFileName,
+    BOOL initializeSEH)
```

Comment:
I don't think we need the option here. Calling the PAL_InitializeDLL in it with FALSE should be fine, I don't see a case where we would need to do it differently.